virtual-desktop/src/app/window-manager/mvd-window-manager/shared/window-manager.service.ts

```diff
@@ -121,17 +121,15 @@ export class WindowManagerService implements MVDWindowManagement.WindowManagerSe
 
   private generateNewWindowPosition(plugin: DesktopPluginDefinitionImpl): WindowPosition {
     let { width: dtWindowWidth, height: dtWindowHeight } = plugin.defaultWindowStyle;
-    const desktopHeight = document.getElementsByClassName('window-pane')[0].clientHeight;
```

Comment:
Can we keep this commented out instead of deleted? Since maybe we'll come back to it (myself, you, or maybe an intern for example) and get a good center-of-screen app spawning method going.